### ISSUE TITLE
pip install version 1.3.2 issue

### ISSUE THREAD

**Issue by lazyrenee** [microsoft/TextWorld#271]
Hi there, 

I have a fair amount of codes written in the past using the `jericho==2.3.0` and I gave up refactoring, when I was trying to reinstall and downgrade to `textworld==1.3.2`, the error shows that `FileNotFoundError: [Errno 2] No such file or directory: 'requirements-vis.txt'`.

Below is the full log output:
```
(base) % pip install textworld==1.3.2
Collecting textworld==1.3.2
  Using cached textworld-1.3.2.tar.gz (637 kB)
    ERROR: Command errored out with exit status 1:
     command: /Users/reneejia/opt/anaconda3/envs/custom-parser/bin/python -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/var/folders/3t/wy0wl0cn13122_mrjp8dz6_40000gn/T/pip-install-k4fipcht/textworld_de3df785e32d43fb8846c78a79f2ec1b/setup.py'"'"'; __file__='"'"'/private/var/folders/3t/wy0wl0cn13122_mrjp8dz6_40000gn/T/pip-install-k4fipcht/textworld_de3df785e32d43fb8846c78a79f2ec1b/setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /private/var/folders/3t/wy0wl0cn13122_mrjp8dz6_40000gn/T/pip-pip-egg-info-u83njwd_
         cwd: /private/var/folders/3t/wy0wl0cn13122_mrjp8dz6_40000gn/T/pip-install-k4fipcht/textworld_de3df785e32d43fb8846c78a79f2ec1b/
    Complete output (5 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/3t/wy0wl0cn13122_mrjp8dz6_40000gn/T/pip-install-k4fipcht/textworld_de3df785e32d43fb8846c78a79f2ec1b/setup.py", line 70, in <module>
        'vis': open('requirements-vis.txt').readlines(),
    FileNotFoundError: [Errno 2] No such file or directory: 'requirements-vis.txt'
    ----------------------------------------
WARNING: Discarding https://files.pythonhosted.org/packages/00/94/f8a5024168724d8cf97eb167ad5802290e9d19f781ecea2a90b43ad8d837/textworld-1.3.2.tar.gz#sha256=c3896ee3717d0f989d605bcbeea667e5646647480669f81fea42947511a0c0f3 (from https://pypi.org/simple/textworld/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
ERROR: Could not find a version that satisfies the requirement textworld==1.3.2 (from versions: 0.0.1, 0.0.3, 0.0.4, 1.0.0, 1.0.1, 1.1.0, 1.1.1, 1.2.0, 1.3.0, 1.3.1, 1.3.2, 1.3.3, 1.3.4rc7, 1.4.0, 1.4.1, 1.4.2, 1.4.3)
ERROR: No matching distribution found for textworld==1.3.2
```
Do you know why this happens?

Thanks in advance!

**Comment by MarcCote** [comment]
That was an oversight on my part. TextWorld v1.3.2 on Pypi doesn't contain that file `requirements-vis.txt`. This has been fixed in textworld==1.3.3.

If 1.3.3 is no good for you, you can always manually git clone the TextWorld repo, switch to 1.3.2, and install it locally. I believe the following would do the trick.
```
git clone https://github.com/microsoft/TextWorld.git -b 1.3.2
pip install ./TextWorld
```

If you need help with refactoring your code to make it compatible with the most recent version of Jericho, let me know. Happy to have a look :).

**Comment by lazyrenee** [comment]
I did try with v1.3.3 but looks like it's not compatible with `jericho 2.3.0`, thanks for the trick, I'll try ~
Also thanks for the help, very much appreciated, I might come up with stupid questions afterwards :(
